### ISSUE TITLE
Introduce maven-module for webjar generation

### ISSUE THREAD

**Issue by hawky-4s-** [camunda/camunda-bpm-spring-boot-starter#155]
Camunda currently only offers a webjar based on the camunda-bpm-webapp without the EE features despite when changing the version to use '-ee'. This leads to confusion because people miss the enterprise features.

(Maybe we could offer an approach how to create an EE webjar?)

**Comment by jangalinski** [comment]
A while ago, we had the maven-assembly script to create the webjar inside this extension. Then we moved it to core, but had always some issues ... it does not seem to produce a syntactically correct webjar, the CE version is not available on maven central and the EE version is not available at all.

It would vote for having a "build the webjar" module inside this starter extension again. Not only would this solve the problem of missing artefacts, it also would allow us an easy way to enhance the ee-cockpit with plugins, something that cannot be done right now. 
I do not think that having "the one and only ee cockpit" available will solve peoples problems, you want to be able to use custom and OSS plugins.

If platform-core wants to follow and provide something similar, we later can argue again if we move it out, but for now, this would be the way to go for milestone 2.0.0.

Basically, this requires the extracted build Bernd already has in the consulting space adopted as a maven-cockpit-plugin-module, so you can do

```
<build>
<plugins>
<plugin>
<artifactId>camunda-spring-boot-webjar-plugin</artifactId>
<configuration>
<version>7.5.4-ee</version>
<cockpit-plugins>
....
</cockpit-plugins>
</configuration>

```
